markhor_control/config/flippers.yaml

```diff
@@ -9,17 +9,17 @@ markhor:
     flipper_fl_position_controller:
       type: position_controllers/JointPositionController
       joint: flipper_fl_j
-      pid: {p: 100.0, i: 0.01, d: 10.0}
+      pid: {p: 0.0, i: 0.0, d: 0.0}
     flipper_fr_position_controller:
       type: position_controllers/JointPositionController
       joint: flipper_fr_j
-      pid: {p: 100.0, i: 0.01, d: 10.0}
+      pid: {p: 100.0, i: 0.0, d: 0.0}
```

Comment:
Is the fact that this one is the only one with a value voluntary? If not, removing the ``pid`` lines would be a better option.